codeforamerica/vita-min
# md502 xml

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-653
## Is PM acceptance required?
- Yes - don't merge until JIRA issue is accepted!
## What was done?
- Added income fields to the MD 502 XML & PDF 
- Attach a blank MD 502R to the XML & PDF if there is income from taxable pensions, IRAs or annuities
## How to test?
- added automated tests that should cover everything
- submit with federal returns with various values in the fields described in the story, and download the output XML & PDFs to make sure the correct values are present
